packages/scandipwa/src/component/MyAccountOrderInformation/MyAccountOrderInformation.style.scss

```diff
@@ -18,6 +18,8 @@
         display: flex;
         justify-content: space-between;
         flex-direction: column;
+        flex-wrap: wrap;
+        row-gap: 25px;
```

Comment:
You have the `row-gap:25px` here and the `row-gap:24px` at the file above.
Is it on purpose? Since I see the inconsistency here.

Reply:
For MyAccountDashboard row-gap is 24px because the column gap is twice more, and for MyAccountOrderInforamation there is no column-gap but title margins are 25px and made the row-gap 25px, is it better to make it 24px? Should I change that?

Reply:
Let's tend to use the same values everywhere. :)
So if there is no particular reason to use gap 25px, please make it the same as other elements have on the page.
Everything should be even.

Reply:
Done!